src/web/app/pages-session/session-submission-page/session-submission-page.component.ts

```diff
@@ -520,7 +520,7 @@ export class SessionSubmissionPageComponent implements OnInit, AfterViewInit {
    */
   saveFeedbackResponses(): void {
     const notYetAnsweredQuestions: Set<number> = new Set();
-    const failToSaveQuestions: Set<number> = new Set();
+    const failToSaveQuestions: Record<string, string> = {};
```

Comment:
```suggestion
    const failToSaveQuestionsError: Record<string, string> = {};
```
Or we should write a comment to explain the map.